### DISCUSSION TITLE
Organizations/Companies/Sites using this gem?

### DISCUSSION THREAD

**Discussion by styd** [styd/apexcharts.rb#34] [Show and tell]
If your organization/company uses ApexCharts.RB in production, please let me know and submit its logo and link here.

**Comment by ollym** [comment]
We do https://ventrata.com

**Comment by styd** [comment]
Great! May I know what kind of data are you visualizing with it? Is this the right logo: https://i1.wp.com/ventrata.com/wp-content/uploads/2018/08/logo_color.png

**Comment by ollym** [comment]
We are visualizing ticket sales for our clients (visitor attractions and tour operators). Yes that logo's right.

**Comment by floriankappert** [comment]
We do https://www.bilendo.de as part of our enterprise dashboards to visualize account receivables data

**Comment by styd** [comment]
Thank you for trusting this gem for your dashboard. I will put bilendo's logo on README soon.

**Comment by styd** [comment]
@floriankappert, do you have a logo that works with white background?
Edit:
I've found it.

**Comment by floriankappert** [comment]
@styd ok thx. Just saw your message, sorry!

**Comment by arferreira** [comment]
Hello, we are also using the gem in our dashboard to optimize revenue, expense and cash flow reports.
https://ecomblue.io

**Comment by styd** [comment]
@arferreira, do you have the logo that will look nice with white background.

**Comment by mb52089** [comment]
We use Apex Charts.  Copient Health
![square_logo](https://user-images.githubusercontent.com/1793762/75354240-abcfd400-587a-11ea-85c1-c47b1aeb8acc.png)

**Comment by mb52089** [comment]
copienthealth.com

**Comment by styd** [comment]
Hello, @mb52089. Do you use apexcharts.js directly or through apexcharts.rb?

**Comment by mb52089** [comment]
apexcharts.rb

using it successfully for several months now.

however I'm currently struggling with a stacked column chart at the
moment.  Does the x value need to be a datetime object?  Right now I'm
trying to use month number and it's not working

On Wed, Feb 26, 2020 at 10:13 AM Adrian Setyadi <notifications@github.com>
wrote:

> Hello, @mb52089 <https://github.com/mb52089>. Do you use apexcharts.js
> directly or through apexcharts.rb?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/styd/apexcharts.rb/issues/34?email_source=notifications&email_token=AANV5YX6QPQVBU2EHC6I753RE2BJZA5CNFSM4ISVQ5Q2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOENATQTA#issuecomment-591476812>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANV5YU6IE2RFZCQFMOPRGTRE2BJZANCNFSM4ISVQ5QQ>
> .
>


-- 
Michael Burke
404.271.8652
LinkedIn <https://www.linkedin.com/in/michael-burke-6418681/>

**Comment by styd** [comment]
@mb52089, what do you mean "month number"? Is it like this "2020-01-01" or this "01"?
In what way that is it not working?
Maybe you can post an issue for that with your code and result.

By the way, what is the use case for apexcharts.rb in your site? I need it to put in the README.md.

**Comment by mb52089** [comment]
the x value (month number) was an integer.  With that as the x value, I got
a completely blank chart.  No axes, no nothing.  When I changed it to a
strftime date representation, it worked fine.  The data hash keys went from
2 (representing February 2019, a year ago) through 12 then back to 1
(representing Jan 2020).
Maybe the order of the integers created a problem since they weren't in the
order you'd expect for an integer?

On Wed, Feb 26, 2020 at 6:51 PM Adrian Setyadi <notifications@github.com>
wrote:

> What do you mean "month number"? Is it like this "2020-01-01" or this "01"?
> In what way that is it not working?
> Maybe you can post an issue for that with your code and result.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/styd/apexcharts.rb/issues/34?email_source=notifications&email_token=AANV5YVPUQ2G2VCHPG5EJXTRE3565A5CNFSM4ISVQ5Q2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOENCK4PQ#issuecomment-591703614>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANV5YSBHF5KC7L7JZNRAX3RE3565ANCNFSM4ISVQ5QQ>
> .
>


-- 
Michael Burke
404.271.8652
LinkedIn <https://www.linkedin.com/in/michael-burke-6418681/>

**Comment by styd** [comment]
@mb52089, you're right. I just tried it. It doesn't show up.
I don't know what happened. The error I got is `Expected number, "translate(Infinity,40)"`
Maybe ApexCharts.JS is trying to convert the x values to something if they are unordered integers.
The quickest fix in your case is to just transform all x values to string.
Maybe you can use `transform_keys`:
 `data_in_hash.transform_keys(&:to_s)`.

But this is still an issue. I need to dig in a little bit more.
Can you post this problem as new issue?
Maybe show some screenshots as well.
Thanks.

**Comment by styd** [comment]
@mb52089 , what do you use apexcharts.rb for in your company?
For example, **bilendo**'s use case is "enterprise dashboards to visualize account receivables data".

**Comment by mb52089** [comment]
We use ApexCharts for reports/charts related to Operating Room utilization
statistics for our Copient Health solution.

On Thu, Feb 27, 2020 at 6:02 PM Adrian Setyadi <notifications@github.com>
wrote:

> @mb52089 <https://github.com/mb52089> , what do you use apexcharts.rb for
> in your company?
> For example, *bilendo*'s use case is "enterprise dashboards to visualize
> account receivables data".
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/styd/apexcharts.rb/issues/34?email_source=notifications&email_token=AANV5YSG4SUK7PNUIWQMLOTRFBA7PA5CNFSM4ISVQ5Q2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOENGJOEA#issuecomment-592221968>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANV5YRXQIF2YXHVLVKFBNDRFBA7PANCNFSM4ISVQ5QQ>
> .
>


-- 
Michael Burke
404.271.8652
LinkedIn <https://www.linkedin.com/in/michael-burke-6418681/>

**Comment by andriesfilmer** [comment]
We proudly use ApexCharts.RB for our volunteers application <https://dutyroster.org> and dutch version <https://inzetrooster.nl>

![Shiftplanner logo](https://shiftplanner.org/assets/shiftplanner-logo-400.png)
![Inzetrooster logo](https://inzetrooster.nl/assets/inzetrooster-logo-400.png)

**Reply by styd** [reply]
Added.

**Comment by styd** [comment]
Hi, @andriesfilmer . Thanks for using apexcharts.rb. Can you tell what you are using it for in both applications?

**Comment by tiagotobias2003** [comment]
[https://lunaltas.com](https://lunaltas.com)

![logo lunaltas](https://user-images.githubusercontent.com/705299/186469992-09234f83-d2cf-49e4-8f31-8f8779fddc36.png)


We are a software house and we always use ApexCharts.RB in our projects.

**Reply by styd** [reply]
> https://lunaltas.com
> 
> ![logo lunaltas](https://user-images.githubusercontent.com/705299/186469992-09234f83-d2cf-49e4-8f31-8f8779fddc36.png)
> 
> We are a software house and we always use ApexCharts.RB in our projects.

Added.
